hummingbot/strategy/liquidity_mining/liquidity_mining.py

```diff
@@ -119,11 +130,17 @@ def tick(self, timestamp: float):
 
     @staticmethod
     def order_age(order: LimitOrder) -> float:
+        """
+        Get the age of a limit order
+        """
         if "//" not in order.client_order_id:
             return int(time.time()) - int(order.client_order_id[-16:]) / 1e6
         return -1.
 
     async def active_orders_df(self) -> pd.DataFrame:
+        """
+        Return the active orders in a DataFram.
```

Comment:
*DataFrame

Reply:
Resolved in commit https://github.com/CoinAlpha/hummingbot/pull/3440/commits/b35ab311e01d62f7b1b468e7d6fb855932b78fa4